pkgs/top-level/all-packages.nix

```diff
@@ -10165,6 +10165,8 @@ with pkgs;
 
   mtail = callPackage ../servers/monitoring/mtail { };
 
+  mu-editor = libsForQt5.callPackage ../applications/editors/mu { };
```

Comment:
Have you tried without `libsForQt5`? So it can be placed under by-name hierarchy?